ci/common/suite.sh

```diff
@@ -1,3 +1,7 @@
+# HACK: get newline for use in strings given that "\n" and $'' do not work.
+NL="$(printf '\nE')"
```

Comment:
This works only as long as you run it on the zero level of indentation. Move NL definition to some function and this is no longer an option.